### PR TITLE
Add colors to network request borders

### PR DESCRIPTION
**Pull request by jonbell-lot23**
Old on left, new on right:
image

We missed some CSS, so I added it

### CODE REVIEW

**Comment by jonbell-lot23** [comment]
Addresses https://linear.app/replay/issue/DES-798/our-left-hand-colors-are-all-screwy

File: src/ui/components/TestSuite/views/TestRecording/TestSectionRow.module.css [hunk 1]
```diff
@@ -31,11 +31,13 @@
 }
 .Row[data-status="success"][data-position="before"],
 .Row[data-type="navigation"][data-position="before"],
+.Row[data-type="network-request"][data-position="before"],
 .Row[data-type="network"][data-position="before"] {
```
Comment (bvaughn) [inline src/ui/components/TestSuite/views/TestRecording/TestSectionRow.module.css:35]:
```suggestion
.Row[data-type="network-request"][data-position="before"] {
```

File: src/ui/components/TestSuite/views/TestRecording/TestSectionRow.module.css [hunk 2]
```diff
@@ -31,11 +31,13 @@
...
   --border-left-color: var(--primary-accent);
 }
 .Row[data-status="success"][data-position="current"],
 .Row[data-type="navigation"][data-position="current"],
+.Row[data-type="network-request"][data-position="current"],
 .Row[data-type="network"][data-position="current"] {
```
Comment (bvaughn) [inline src/ui/components/TestSuite/views/TestRecording/TestSectionRow.module.css:41]:
```suggestion
.Row[data-type="network-request"][data-position="current"] {
```

**Review by bvaughn — APPROVED** [review]
Looks like "network" was an outdated enum.

**Pull request merged**